v20.1/inverted-indexes.md

```diff
@@ -83,7 +87,15 @@ Tables are not locked during index creation thanks to CockroachDB's [schema chan
 Indexes create a trade-off: they greatly improve the speed of queries, but slightly slow down writes (because new values have to be copied and sorted). The first index you create has the largest impact, but additional indexes only introduce marginal overhead.
 
 ### Comparisons
-Currently, inverted indexes only support equality comparisons using the `=` operator. If you require comparisons using `>`, `<=`, etc., you can create an index on a computed column using your JSON payload, and then create a regular index on that. So if you wanted to write a query where the value of "foo" is greater than three, you would:
+
+#### JSONB
+
+Inverted indexes on `JSONB` columns support the following comparison operators:
+
+- "is contained by": [`<@`](functions-and-operators.html#operator-is-contained-by)
+- "contains": [`@>`](functions-and-operators.html#operator-contains)
```

Comment:
Crap, I actually forgot about a case. You can also use equality, but only when you've reached into the json document with the `->` operator. It's equivalent to using `@>`. Here's an example:

```
select * from a where j @>  '{"foo": "1"}';
```

This is equivalent to:

```
select * from a where j ->'foo' = '"1"';
```

I can provide more examples if you like.

@RaduBerinde are there more operators that are currently accelerated using inverted indexes?

Reply:
No, as far as I can tell, only `@>` is (and we have rules that convert other expressions to use this operator when possible, like `j ->'foo' = '"1"'` to `j @> '{"foo": "1"}'`).

Reply:
Cool!  Updated this section to note the possibility of using `=` in this specific case, and to add both of these examples to show that it's really just `@>` again